lib/db_connection.ex

```diff
@@ -717,7 +717,7 @@ defmodule DBConnection do
           {result, run(conn, &run_status/3, nil, opts)}
         catch
           kind, error ->
-            stacktrace = System.stacktrace()
+            stacktrace = __STACKTRACE__
```

Comment:
With the `__STACKTRACE__` construct we don't need to capture it in a variable anymore, we can inline it - there's no risk of capturing the wrong stacktrace like with `System.stacktrace/0`

Reply:
Thanks, will fix!